lib/resolver.fallback.js

```diff
@@ -19,14 +22,14 @@ export default class PodletClientFallbackResolver {
     #log;
     #metrics;
     #histogram;
-    #httpAgent;
-    #httpsAgent;
+    // #httpAgent;
+    // #httpsAgent;
```

Comment:
Undici seems happy to handle http and https interchangeably out of the box but you mentioned we may want to use 2 separate Undici client instances, 1 for http and 1 for https @trygve-lie ?

Reply:
If it can, then thats great :) Less complexity for us.